style.css

```diff
@@ -674,6 +674,7 @@ label {
     display: block;
     color: #fff;
     margin: 10px 0 -5px 0;
+    text-align: start;
   }
```

Comment:
1. Updated the form labels display which were initially only placeholder. This is to help with accessibility through website text to voice conversion software. 
2. Also changed text color from black to while to improve readability of labels and contrast.